crates/swc_ecma_parser/tests/tsc/multipleDefaultExports03.ts

```diff
@@ -4,5 +4,5 @@
 export default class C {
 }
 
-export default class C {
```

Comment:
Please revert this

Reply:
`tsc` test suite is **copied** tests so we should not modify the input

Reply:
but the duplicated-binding error  report before the export default twice
so i need to update the expect error to dup-binding error ?

Reply:
Yes

Reply:
done